browser/brave_browser_process_impl.cc

```diff
@@ -336,6 +337,20 @@ BraveBrowserProcessImpl::request_otr_component_installer() {
 }
 #endif
 
+reduce_language::ReduceLanguageComponentInstallerPolicy*
+BraveBrowserProcessImpl::reduce_language_component_installer() {
```

Comment:
we really need to move away from this pattern of adding an accessor on `g_brave_browser_process` for all of this stuff. The normal pattern is to define a `Register*Component` in `*_component_installer` file. Call the method here https://github.com/brave/brave-core/pull/19865/files#diff-cbb2c9ef9feda283a85ebfea03e43b88e6c4ef56523518df6f5ff7caf50a524aR57 and pass in the observer so it would be `RegisterReduceLanguageComponent(g_browser_process->local_data_files_service(), reduce_language_service)` - but I'm a bit confused because I don't see where the component installer gets registered with the component update service?